deploy/webhook/member-operator-webhook.yaml

```diff
@@ -0,0 +1,107 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: member-operator-webhook
+objects:
+- apiVersion: scheduling.k8s.io/v1
+  kind: PriorityClass
+  metadata:
+    name: sandbox-users-pods
+    labels:
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  value: -10
+  globalDefault: false
+  description: "Priority class for pods in users' namespaces"
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: member-operator-webhook
+    namespace: ${NAMESPACE}
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  spec:
+    ports:
+    - port: 443
+      targetPort: 8443
+    selector:
+      app: member-operator-webhook
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: member-operator-webhook
+    namespace: ${NAMESPACE}
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: member-operator-webhook
+    template:
+      metadata:
+        name: member-operator-webhook
+        labels:
+          app: member-operator-webhook
+      spec:
+        containers:
+        - name: mutator
+          image: ${IMAGE}
+          command:
+          - member-operator-webhook
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 500m
+              memory: 500Mi
+            requests:
+              cpu: 250m
```

Comment:
Isn't it too much for the cpu request? We can tweak it later though.

Reply:
Yeah, probably. I didn't spend much time with these values TBH :sweat_smile: :man_shrugging: 
I could have set something smaller - in an example that I saw there was:
```
            limits:
              cpu: 500m
              memory: 128Mi
            requests:
              cpu: 250m
              memory: 64Mi
```
so, I just blindly increased both vales, but it's true that I could keep the CPU smaller

Reply:
decreased here 7b5a7ff